### ISSUE TITLE
Change Username

### ISSUE THREAD

**Issue by neale** [keybase/keybase-issues#1576]
Is it possible yet to change the username? I made the account a while back and just now pushed the key. I don't want that user name that I made back then, it'd be way rad. 

Thanks,

**Comment by zQueal** [comment]
No, this is currently not possible. We've had this question before and I'm pretty sure the solution was to simply re invite yourself for an account.

Make use of the built in Gmail email proxy if you happen to have Gmail, as you won't be able to send an invite to your old email address again. `username+keybase@gmail.com` should work.

A bit cumbersome to figure out, but it should work out for you.

**Comment by neale** [comment]
Alright, Thanka a lot!
